### DISCUSSION TITLE
LCSC parts

### DISCUSSION THREAD

**Discussion by moto25** [revk/xESP32-Faikout#180] [General]
Hi, i am trying to order pcb from LCSC and your bom includes 2 parts which they dont have stocked. 
C5137770 and C2684786. One is step down converter, other is fuse. Any advice with what parts it could be replaced? I initially wanted to order from Amazon but they dont ship to my country. Thank you

**Comment by revk** [comment]
Not a fuse, an ESP protection diode, you could get away with non-fit on that.

The regulator, no, not sure. I spent ages finding a regulator after a previous one was not available, and it ties in with the associated passives and layout, so not sure if there is a slot in replacement, sorry. If you find one, let me know, as I am not having to pre-order these.

**Reply by moto25** [reply]
Wow that was a quick reply. LCSC recommended this part, would that work?
https://www.lcsc.com/product-detail/DC-DC-Converters_Texas-Instruments-TPS560430XDBVR_C524782.html

**Reply by revk** [reply]
Well the pinout matches... It may work.

**Reply by moto25** [reply]
will try, thank you

**Reply by MUN0X** [reply]
Fyi, i ordered the boards with the TI step down converter. It doesnt work…
The esp doesn’t receive any power. If I feed the esp directly via usb it works fine and i can control the AC.

**Reply by revk** [reply]
That is surprising... The pin out looked sensible. Have you checked voltages?
